sample-points.csv

```diff
@@ -0,0 +1,100 @@
+47.6205,-122.3493
```

Comment:
Will this grow into git-lfs size?

Reply:
Should be okay, but maybe we can talk about this more?